src/control/server/config/server.go

```diff
@@ -289,8 +296,9 @@ func DefaultServer() *Server {
 		Hyperthreads:    false,
 		Path:            defaultConfigPath,
 		ControlLogMask:  common.ControlLogLevel(logging.LogLevelInfo),
-		EnableVMD:       false, // disabled by default
-		EnableHotplug:   false, // disabled by default
+		EnableVMD:       false,      // disabled by default
+		EnableHotplug:   false,      // disabled by default
+		CoreDumpFilter:  0b00010011, // private, shared, ELF
```

Comment:
Would be nice to add a comment here pointing to the manpage mentioned in the config file